_posts/wiki/2020-03-23-robotic-skin.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robotic Skin
+description: Research Page
+tags: [robotics,panda,simulator,research]
+permalink: robotic_skin.html
+---
+
+{% include image.html url="research/skin_unit.png" max-width="100%" description="Flexible skin unit created by etching the circuit diagram into a copper sheet and then encasing it in the flexible polymer ecoflex." %}
```

Comment:
This "skin unit" thing you keep talking about is not a standard concept, nor it doesn't make much sense. If you have a whole-body skin, what does it mean to have a skin unit? I don't understand why you refer to this as skin unit as if it was as mainstream as the word pixel or gradient descent. I would rephrase parts of this document and using those two words as common words rathrr than specific official words (and this is true also for the paper)

Reply:
For what concerns this caption: you should also add a description of the capability, which is missing as of now.